spec/services/sdr/repository_spec.rb

```diff
@@ -149,6 +149,59 @@
     end
   end
 
+  describe '#latest_user_version' do
+    context 'when the object is found' do
+      let(:object_client) { instance_double(Dor::Services::Client::Object, user_version: user_version_client) }
+      let(:user_version_client) { instance_double(Dor::Services::Client::UserVersion, inventory:) }
```

Comment:
There's really no reason to use instance doubles for `Dor::Services::Client::UserVersion`.